.ocamlformat

```diff
@@ -1,2 +1 @@
-version=0.24.1
```

Comment:
Why have you removed the version?

Reply:
Having a fixed version is good for formatting consistency, but implies having to fix the OCamlformat version at dependency level.

So there are two solutions:
- fix OCamlformat in the dune file
- let any version of OCamlformat work (which is the case here)

I can do what you want.

Reply:
If we don't fix the version, every developer would have different formatting, so most PRs would contain formatting changes as well, right?

Reply:
Yes

Reply:
And you think that's better than synching the ocamlformat version between all devs?